core/v0/api/core.yaml

```diff
@@ -1943,6 +1940,17 @@ components:
         time:
           $ref: '#/components/schemas/Time'
 
+    User: 
```

Comment:
User information strictly remains with the BAP and must only be accessed by the BPP via a consent framework like DEPA. In face, the personal details like name, gender, contact info `Person` schema should also be removed from the Core spec and should only allow an ID with a consent token to be transmitted that authorizes the BPP to fetch specific details from the BAP.